regulations/static/components/tooltips/CopyBtn.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+    <div class="copy-btn-container">
+        <button
+            class="copy-btn text-btn"
+            :class="buttonClasses"
+            :aria-label="btn_type === 'icon' ? ariaLabel : false"
```

Comment:
this may be my inexperience with vue, but this seems odd to me.  If the btn_type is not "icon" the aria label is "false"?  also, why spread out the logic into 2 places?  

```javascript
        :aria-label="ariaLabel"
        ...
        ariaLabel() {
            return btn_type === 'icon' ? `${this.label} for ${this.title}`: '';
        }

```

Reply:
Fixed.